scenarios/sslstream.benchmarks.yml

```diff
@@ -0,0 +1,99 @@
+# Example:
+# crank --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/sslstream.benchmarks.yml --scenario read-write --profile aspnet-perf-lin
+
+imports:
+  - https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/aspnet.profiles.standard.yml
+
+variables:
+  # common parameters
+  serverPort: 5000
+  sendBufferSize: 32768 # bytes
+  receiveBufferSize: 32768 # bytes
+  sslProtocols: "None" # <Default|None|Ssl2|Ssl3|Tls|Tls11|Tls12|Tls13>

```

Comment:
What happens if it's "None"? Does it refuse to connect, or is it not encrypted? Is this multi-value?

Reply:
(should default be "Default"? 😄)

Reply:
The naming is a bit unfortunate and has historical baggage, but Default is actually `Ssl3 | Tls` and is deprecated. I will see if I can remove it from the list.

https://github.com/dotnet/runtime/blob/77d643d8678d0cae1144557f106bb693a1cbca25/src/libraries/System.Net.Primitives/src/System/Net/SecureProtocols/SslEnumTypes.cs#L10-L26

Reply:
the None (= 0) value lets OS decide (i.e. highest supported TLS version)

Reply:
We don't have to use the actual SslProtocols enum here. It might be beneficial to limit the values to the ones we/endusers actually use. I.e. use a new enum that is later mapped to SslProtocols (esp given that you have an explicit binder anyway). And I'd suggest then not to use the confusing naming like "None" and "Default" 😅 So only have a small, like, 2-3 value enum here.

(btw on a side note, i don't think that letting OS decide stuff is a good thing from benchmarking perspective, it's not reproducible enough 😅 it's better to set everything explicitly 🤔)